docs/getting-started/simple.rst

```diff
@@ -127,10 +127,7 @@ to each PyVista data object:
     from pyvista import examples
 
     mesh = examples.load_airplane()
-    mesh.plot(screenshot='airplane.png')
-
-
-.. image:: ../images/auto-generated/airplane.png
```

Comment:
This part is not working.
https://dev.pyvista.org/getting-started/simple.html#plotting
![image](https://user-images.githubusercontent.com/7513610/115156232-31859300-a0be-11eb-80d4-24975e344b97.png)

Reply:
Fixed in https://github.com/pyvista/pyvista/pull/1238/commits/d0dca76fb98fd8fb6f35d0844a02259cd7a456dc.  Issue is our default backend changed to `ipyvtklink`, which is incompatible with sphinx jupyterlab.